test/integration/grpcroute_test.go

```diff
@@ -65,6 +65,8 @@ func grpcEchoResponds(ctx context.Context, url, hostname, input string) error {
 }
 
 func TestGRPCRouteEssentials(t *testing.T) {
+	// TODO: implement translator for grpc routes
```

Comment:
Can we add an issue for this and link it here?

Reply:
added:#3752